section4Code/target/classes/logback.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
```

Comment:
Anything under target no need submit as they are generated when compile, pls remove

Reply:
Same apply to below jar file